packages/entity-browser/src/modules/entityBrowser/reducer.js

```diff
@@ -14,14 +14,24 @@ const clearRecordStore = state => ({
   recordStore: {}
 })
 
+const setSearchTerm = (state, {payload}) => {
+  const term = payload.searchTerm
```

Comment:
why do you pass an object containing a name and value and only set the value in the state? i would either save the whole object to the state (and then use the singleTransferRecuder) is name attribute is needed or only pass value (and use singleTransferReducer)